packages/ihe-gateway-sdk/src/client/ihe-gateway.ts

```diff
@@ -41,11 +41,15 @@ export class IHEGateway {
    * Patient Discovery (XCPD ITI-55) request.
    * https://profiles.ihe.net/ITI/TF/Volume2/ITI-55.html
    *
-   * @param patientDiscoveryRequest A patient discovery transaction request to Ihe Gateway.
+   * @param pdRequest A patient discovery transaction request to IHE Gateway.
    *
    */
-  async startPatientDiscovery(patientDiscoveryRequest: PatientDiscoveryRequest): Promise<void> {
-    await this.api.post(IHEGateway.PATIENT_DISCOVERY_ENDPOINT, patientDiscoveryRequest);
+  async startPatientDiscovery(pdRequest: PatientDiscoveryRequest[]): Promise<void> {
+    await this.api.post(IHEGateway.PATIENT_DISCOVERY_ENDPOINT, pdRequest, {
```

Comment:
Ideally we would make input/outputs of the API always a JSON object, not an array. It allows to add more properties later - and we prob wouldn't need to set the JSON header. :)